scripts/sanitycheck

```diff
@@ -522,7 +523,6 @@ class BinaryHandler(Handler):
         """
         super().__init__(instance, type_str)
 
-        self.valgrind = False
         self.terminated = False
```

Comment:
You are leaving more dead code around :) (there is no use of this self.terminated anymore)
(just a remark, you don't necessarily need to do anything)

Reply:
I am still not done, as you can see, CI is still failing because of this change. Getting back to this now.